packages/gestalt/src/Contents.js

```diff
@@ -202,6 +204,37 @@ class Contents extends Component<Props, State> {
     }
   };
 
+  /**
+   * Identify when is required to repositioning the popover, i.e., the windows popover has 90% of
+   * viewport (available screen height), and set the controller variable as `true`;
+   * When the height of popover is less than 90% the controller variable is `false`;
+   */
+  balancePopoverPosition(): PopoverOverride {
+    // Required because of SSR
+    if (!window || !document) {
+      return {};
```

Comment:
Consider instead returning `null` here, and either using nullish coalescing to an empty object on line 249, or conditionally spreading `overridePropsToMaxPopoverSize` on line 267